example/src/components/FocusedParticipant.tsx

```diff
@@ -0,0 +1,61 @@
+import { BrandColors, AdditionalColors } from '@colors';
+import * as Membrane from '@jellyfish-dev/react-native-membrane-webrtc';
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+
+import { Participant } from './NotFocusedParticipants';
+import { RoomParticipant } from './RoomParticipant';
+import { StopScreencastingWithFocus } from './StopScrencastingWithFocus';
+
+type FocusedParticipantProp = {
+  focusedParticipant: Participant;
+  onPress: (string) => void;
+};
+
+export const FocusedParticipant = ({
+  focusedParticipant,
+  onPress,
+}: FocusedParticipantProp) => {
+  const focusedTrack = focusedParticipant.participant.tracks.find(
+    (t) => t.id === focusedParticipant.trackId
+  );
+
+  return (
+    <View style={styles.focusedParticipantContainer}>
+      <View style={styles.focusedParticipant}>
```

Comment:
Simple copy pasting won't work here, so I think it would be best to leave it as it is for now especially since MV-395 will be reworking it's styling anyway.